deployment/ccip/changeset/internal/deploy_home_chain.go

```diff
@@ -185,10 +186,61 @@ func BuildSetOCR3ConfigArgs(
 	return offrampOCR3Configs, nil
 }
 
+type MultiOCR3BaseOCRConfigArgsSolana struct {
+	ConfigDigest                   [32]byte
+	OcrPluginType                  uint8
+	F                              uint8
+	IsSignatureVerificationEnabled bool
+	Signers                        [][20]uint8
```

Comment:
These are the Solana onchain signing keys right? Even though `uint8` and `byte` are technically the same, `byte` is used where we're referring to actual byte arrays / slices to make it clear that this is what is being represented.